fuzzinator/call/file_reader_decorator.py

```diff
@@ -43,4 +52,7 @@ def call(self, cls, obj, *, test, **kwargs):
                 issue['test'] = f.read()
             issue['filename'] = os.path.basename(test)
 
+        if self.cleanup:
+            os.remove(test)
```

Comment:
Can we have a test for this somehow? An input file copied into a temp directory and then checked for its existence after a call?